mutagen/flac.py

```diff
@@ -519,7 +519,7 @@ def write(self):
                 track_flags |= 0x40
             track_packed = struct.pack(
                 self.__CUESHEET_TRACK_FORMAT, track.start_offset,
-                track.track_number, track.isrc, track_flags,
+                track.track_number, track.isrc or b"\0", track_flags,
```

Comment:
Looks good to me, but is writing a null byte instead of the ISRC documented somewhere?

Reply:
In the FLAC spec, write 12 null bytes is the correct representation when the ISRC is absent. Packing a single null byte with `struct` will produce 12 null bytes.